### PR TITLE
docs(atomic-angular): add explanation on initialization process

### PR DESCRIPTION
**Pull request by olamothe**
https://coveord.atlassian.net/browse/KIT-1455

### CODE REVIEW

File: packages/atomic-angular/README.md [hunk 1]
```diff
@@ -36,6 +36,43 @@ export class AppModule {}
 
 Once this is done, you will be able to reference all atomic components inside that module.
 
+## Initializing the search interface
```
Comment (jpmarceau) [inline packages/atomic-angular/README.md:39]:
```suggestion
## Initializing the Search Interface
```

File: packages/atomic-angular/README.md [hunk 2]
```diff
@@ -36,6 +36,43 @@ export class AppModule {}
...
+
+You can initialize the search interface at any time you desire in the lifecycle of your application. One such suitable lifecyle hook is the `AfterViewInit`,
```
Comment (jpmarceau) [inline packages/atomic-angular/README.md:41]:
```suggestion
You can initialize the search interface at any time in the lifecycle of your application. One such suitable lifecyle hook is `AfterViewInit`.
```

File: packages/atomic-angular/README.md [hunk 3]
```diff
@@ -36,6 +36,43 @@ export class AppModule {}
...
+
+For example, using the above `AppModule` as a starting point, which bootstrap `AppComponnent`:
```
Comment (jpmarceau) [inline packages/atomic-angular/README.md:43]: I don't really see what you mean here: "which bootstrap `AppComponnent`" ?

Comment (olamothe) [reply packages/atomic-angular/README.md:43]: If you look at the full README, in the `AppModule` code, there's a `bootstrap` configuration

File: packages/atomic-angular/README.md [hunk 4]
```diff
@@ -36,6 +36,43 @@ export class AppModule {}
 
 Once this is done, you will be able to reference all atomic components inside that module.
 
+## Initializing the Search Interface
+
+You can initialize the search interface at any time in the lifecycle of your application. One such suitable lifecyle hook is `AfterViewInit`.
+
+For example, using the above `AppModule` as a starting point, which bootstrap `AppComponent`:
```
Comment (jpmarceau) [inline packages/atomic-angular/README.md:43]: Oh I think I see now, thanks!
```suggestion
The following example uses this `AfterViewInit` hook, and relies on the above `AppModule` as a starting point, which bootstraps `AppComponent`.
```

**Review by jpmarceau — APPROVED** [review]

**Review by ThibodeauJF — APPROVED** [review]

**Pull request merged**